dataset/src/main/java/org/kitesdk/examples/data/GenerateEvents.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright 2015 Cloudera, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.kitesdk.examples.data;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Random;
+import java.util.UUID;
+import org.apache.avro.util.Utf8;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.util.ToolRunner;
+import org.apache.hadoop.conf.Configured;
+import org.apache.hadoop.util.Tool;
+import org.kitesdk.data.DatasetDescriptor;
+import org.kitesdk.data.DatasetWriter;
+import org.kitesdk.data.Datasets;
+import org.kitesdk.data.View;
+import org.kitesdk.data.event.StandardEvent;
+
+public class GenerateEvents extends Configured implements Tool {
+  protected Random random;
+  protected long baseTimestamp;
+  protected long counter;
+
+  public GenerateEvents() {
+    random = new Random();
+  }
+  @Override
+  public int run(String[] args) throws Exception {
+    long counter = 0l;
+    baseTimestamp = System.currentTimeMillis();  
+
+    View<StandardEvent> events = Datasets.load(
+        (args.length==1 ? args[0] : "dataset:hive:events"), StandardEvent.class);
```

Comment:
I noted this elsewhere, but I think it would be better to use a variable rather than the inline test here.